packages/fetch-api/src/normalize/gif.ts

```diff
@@ -28,15 +28,10 @@ type Tag = { text: string }
 // tags sometimes are objects that have a text prop, sometimes they're strings
 const getTag = (tag: Tag | string) => (typeof tag === 'string' ? (tag as string) : (tag as Tag).text)
 
-const normalize = (gif: any, responseId: string = '', pingbackType: PingbackEventType = '') => {
+const normalize = (gif: any) => {
     const newGif: IGif = { ...gif }
     newGif.id = String(newGif.id)
     newGif.tags = (newGif.tags || []).map(getTag)
-    if (!newGif.bottle_data) {
-        newGif.bottle_data = {} as any
-    }
-    newGif.response_id = responseId
```

Comment:
We no longer need to manually update our gifs, `response_id` is already on the gif object in the `analytics_response_payload`, and so is the `pingback_event_type`